src/hooks/useReviewDuplicatesNavigation.tsx

```diff
@@ -13,15 +13,15 @@ function useReviewDuplicatesNavigation(stepNames: string[], currentScreenName: S
 
     useEffect(() => {
         if (currentScreenName === 'confirmation') {
-            setPrevScreen(intersection[intersection.length - 1] ?? '');
+            setPrevScreen(intersection.length > 0 ? intersection.at(-1) : undefined);
```

Comment:
I guess it will be the same
```suggestion
            setPrevScreen(intersection.at(-1));
```
![image](https://github.com/user-attachments/assets/aad435c0-188c-49e0-af9a-ddda142ffced)